src/client/public/config/openfin/launcher-demo.json

```diff
@@ -3,11 +3,12 @@
     "name": "Reactive Ecosystem Launcher",
     "url": "https://web-demo.adaptivecluster.com/launcher",
     "uuid": "reactive-launcher-demo",
-    "defaultWidth": 650,
-    "defaultHeight": 52,
+    "defaultWidth": 400,
+    "defaultHeight": 120,
```

Comment:
I wouldn't actually change these JSON files in this way. Actual installed applications without these changes point to this file.

Reply:
The right place to change the config is here: https://github.com/AdaptiveConsulting/ReactiveTraderCloud/blob/master/src/node-server/openfin-config/src/config/launcher.ts

As this config is now served as part of the environment.

Unfortunately the local config does not follow this pattern yet, so please change only launcher-local.json, and the config in the location above.

Reply:
The above comments apply to all the configs for dev, uat, demo etc.